diagnostic.md

```diff
@@ -3,67 +3,57 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#start-of-content (id)
+#js-flash-container (id)
+.container site-footer-container (class)
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+1. $("li")
+2. $("ol" > "li")
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$(".listerItem").closest("form")
 
 ## Question 4
 
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+Getter: find(), attr()
+Setter: each(), append()
```

Comment:
So our setter methods typically take an argument because that is what it will set the node to. For example we could use `.innerHtml()` to get the inner HTML, and `innerHtml('Danielle')` to set the inner HTML to the string 'Danielle'